src/compiler/factory.ts

```diff
@@ -339,20 +339,19 @@ namespace ts {
     }
 
     export function createPropertyAccess(expression: Expression, name: string | Identifier, location?: TextRange) {
-        return createPropertyAccessWithDotToken(expression, createSynthesizedNode(SyntaxKind.DotToken), name, location, /*flags*/ undefined);
+        return createPropertyAccessWithDotToken(expression, name, location, /*flags*/ undefined);
     }
 
-    export function createPropertyAccessWithDotToken(expression: Expression, dotToken: Node, name: string | Identifier, location?: TextRange, flags?: NodeFlags) {
+    export function createPropertyAccessWithDotToken(expression: Expression, name: string | Identifier, location?: TextRange, flags?: NodeFlags) {

```

Comment:
Just collapse this and the above definition and update all references to just call `createPropertyAccess`.